### ISSUE TITLE
Re: Receipt Theming

### ISSUE THREAD

**Issue by i-am-chibuzo** [fortune2009/e-store#3]
Oga fortune, The class is not supposed to be written as a test... You were supposed to write a utility class dedicated for reciept theming and write your unit tests to ensure the class works.

**Comment by fortune2009** [comment]
Yes! I agree with you boss. So I usually test my algorithm before writing the utility class and methods. I do this to ensure the computer and I are in agreement before I refactor and run the utility test.

Since you have identified this as an issue, I will correct it and ensure I follow due unit test procedures.

**Comment by fortune2009** [comment]
issue unresolved

**Comment by i-am-chibuzo** [comment]
Oya go and resolve it so that i can close it kia kia
